test/reader_test.rb

```diff
@@ -855,6 +855,7 @@ class ReaderTest < Minitest::Test
       end
 
       test 'unreadable file referenced by include directive is replaced by warning' do
+        next if windows? # JRuby on Windows runs this even with the conditional on the block
```

Comment:
Is there a reason why you're not using Rspec `skip``? It allows to specify message and makes it clearly visible in test results which tests were skipped.

Reply:
Sure, I could do that too. Thanks for the idea. I'm just focusing on getting the test suite to pass. Perfection is not my goal right now.

Reply:
I either need to switch away from IO.popen or just disable the CLI tests that are failing. The reason I'm using IO.popen is because there was a bug in popen3 that was causing the test to fail in some version of JRuby. But that could be outdated now. The point is, I didn't pick stuff randomly. Getting these CLI tests to work on all Rubies and all platforms takes hours, if not days. And I hate it every time I have to touch them.